CONTRIBUTING.md

```diff
@@ -4,19 +4,21 @@
 
 ### Installing
 
-Once the repo has been cloned, install Lerna:
+Once the repo has been cloned, install Lerna
 
 ```bash
 yarn global add lerna
 ```
 
-Install all package dependencies:
+Install all package dependencies and build all workspaces (local dependencies)
 
 ```bash
-yarn install
+yarn
```

Comment:
I'm not a really a fan of the implicit install in documentation, seems like it might make it more confusing for people who don't know that running yarn by itself does the install.

Reply:
@microsoft/fast-dna-collaborators please provide guidance.  I don't disagree, but, it was my understanding we were documenting to support the experienced user to start.